modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -0,0 +1,276 @@
+----
+<1> Specifies the prefix to include in the {cluster-manager} role name. The default is `ManagedOpenShift`.
+<2> Optional: Specifies a permissions boundary Amazon Resource Name (ARN) for the role. For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] in the AWS documentation.
+<3> Selects the role creation mode. You can use `auto` mode to automatically create the {cluster-manager} role and link it to your {cluster-manager} organization account.
+<4> Links the {cluster-manager} role to your {cluster-manager} organization account.
+.. Select *Next* on the {cluster-manager} *OCM role* page.
+.. On the *User role* page, click the copy button for the *User role* and run the command in the CLI. Follow the prompts to create the user role:
++
+[source,terminal]
+----
+$ rosa create user-role
+----
++
+.Example output
+[source,terminal]
+----
+I: Creating User role
+? Role prefix: ManagedOpenShift <1>
+? Permissions boundary ARN (optional): <2>
+? Role creation mode: auto <3>
+I: Creating ocm user role using 'arn:aws:iam::<aws_account_id>:user/<aws_username>'
+? Create the 'ManagedOpenShift-User-<ocm_username>-Role' role? Yes
+I: Created role 'ManagedOpenShift-User-<ocm_username>-Role' with ARN 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-User-<ocm_username>-Role'
+I: Linking User role
+? User Role ARN: arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-User-<ocm_username>-Role
+? Link the 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-User-<ocm_username>-Role' role with account '<ocm_user_account_id>'? Yes <4>
+I: Successfully linked role ARN 'arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-User-<ocm_username>-Role' with account '<ocm_user_account_id>'
+----
+<1> Specifies the prefix to include in the user role name. The default is `ManagedOpenShift`.
+<2> Optional: Specifies a permissions boundary Amazon Resource Name (ARN) for the role. For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] in the AWS documentation.
+<3> Selects the role creation mode. You can use `auto` mode to automatically create the user role and link it to your {cluster-manager} user account.
+<4> Links the user role to your {cluster-manager} user account.
+.. On the {cluster-manager} *User role* page, select *Ok*.
+.. Under the *Accounts and roles* page, verify that your AWS account is listed as an *Associated AWS account*. 
+
+. If the required AWS IAM *Account roles* are not automatically detected and listed on the *Accounts and roles* page, create the roles and policies:
+.. Click the copy buffer next to the `rosa create account-roles` command. Run the command in the CLI to create the required AWS account-wide roles and policies, including the Operator policies::
++
+[source,terminal]
+----
+$ rosa create account-roles
+----
++
+.Example output
+----
```

Comment:
The other example output blocks used source,terminal. Suggest adding here too for consistency

Reply:
Well spotted, thank you. Resolved.